pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -1651,7 +1649,7 @@ var _ = Describe("domXMLWithoutKubevirtMetadata", func() {
 		mockDomain.EXPECT().GetXMLDesc(libvirt.DOMAIN_XML_MIGRATABLE).MaxTimes(1).Return(string(domXML), nil)
 		newXML, err := domXMLWithoutKubevirtMetadata(mockDomain, vmi)
 		Expect(err).To(BeNil())
-		Expect(newXML).To(Equal(expectedXML))
+		Expect(newXML).To(MatchXML(expectedXML))
```

Comment:
Why this changed? (and at linne 1638)

Reply:
The test was failing locally, I am running them directly with `go test ./pkg/virt-launcher/virtwrap/`, so this fix the issue and also make sense.

Reply:
wait... didn't see the new line :-(